pages/web/hosting/sql_how_to_migrate_database_from_sql_4_to_mysql_55/guide.fr-fr.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Comment migrer votre base de donnees de SQL 4 vers MySQL 5.6
-slug: mutualise-comment-migrer-votre-base-de-donnees-de-sql-4-vers-mysql-55
+title: Comment migrer votre base de donnees de MySQL 4 vers MySQL 5.6
+slug: mutualise-comment-migrer-votre-base-de-donnees-de-mysql-4-vers-mysql-56
```

Comment:
Don't change the slug! It will change the url sent to the customer...

Reply:
Done